profiler/CMakeLists.txt

```diff
@@ -53,12 +53,11 @@ set (srcs
   MAPL_Profiler.F90
 
   )
-esma_add_library (${this} SRCS ${srcs} DEPENDENCIES gftl-shared gftl MAPL.shared MPI::MPI_Fortran)
+esma_add_library (${this} SRCS ${srcs} DEPENDENCIES gftl-shared MAPL.shared MPI::MPI_Fortran)
+target_link_libraries(${this} INTERFACE gftl-shared)
```

Comment:
Why is this line needed?   It is already linked in the line above.

Reply:
Agreed, this line is not needed. Implemented in 3510c1a.